docs/source/changelog/misc/pick_free_port.rst

```diff
@@ -0,0 +1,6 @@
+[Misc] Server launcher picks free port if default/choice in use
+===============================================================
+
+* The function launching the `libertem-server` application will now
+  pick a free, random port if the default or chosen port is in use.
+  Closes #1184.
```

Comment:
Nitpick: If you use :issue:\`1184\` here, it will link directly to the issue

Reply:
Made that change, thanks!